controllers/users.js

```diff
@@ -201,6 +201,23 @@ const getUsernameAvailabilty = async (req, res) => {
   }
 };
 
+const getUsername = async (req, res) => {
+  try {
+    const { firstname, lastname, dev } = req.query;
+    if (dev === "true") {
+      const username = await userQuery.generateUniqueUsername(firstname, lastname);
+      return res.json({ username });
+    } else {
+      return res.status(404).json({
+        message: "Data Not Found",
```

Comment:
message: "UserName Not Found"

Reply:
Done Can you please check latest commit changes